Sources/DistributedActors/Timers+Distributed.swift

```diff
@@ -1,167 +0,0 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the Swift Distributed Actors open source project
-//
-// Copyright (c) 2018-2022 Apple Inc. and the Swift Distributed Actors project authors
-// Licensed under Apache License v2.0
-//
-// See LICENSE.txt for license information
-// See CONTRIBUTORS.md for the list of Swift Distributed Actors project authors
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-//===----------------------------------------------------------------------===//
-
-import Dispatch
-import Distributed
-import DistributedActorsConcurrencyHelpers
-import struct NIO.TimeAmount
-
-struct DistributedActorTimer {
-    @usableFromInline
-    let key: TimerKey
-    @usableFromInline
-    let repeated: Bool
-    @usableFromInline
-    let handle: Cancelable
-}
-
-struct DistributedActorTimerEvent {
-    let key: TimerKey
-    let owner: ActorID
-}
-
-/// Creates and manages timers which may only be accessed from the actor that owns it.
-///
-/// _BehaviorTimers are bound to this objects lifecycle, i.e. when the actor owning this object is deallocated,
-/// and the `ActorTimers` are deallocated as well, all timers associated with it are cancelled.
-@available(*, deprecated, message: "Actor timers cannot participate in structured cancellation, and will be replaced with patterns using swift-async-algorithms (see Timer)")
-public final class ActorTimers<Act: DistributedActor> where Act.ActorSystem == ClusterSystem {
```

Comment:
bye bye 👋 If we ever get some other type similar to this it'll be based on tasks... we'll see if it ever comes back